packages/whenable/test/test-watch.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+import test from 'ava';
+
+import { makeHeapZone } from '@agoric/base-zone/heap.js';
+import { prepareWhenableModule } from '../src/module.js';
+
+/**
+ * @param {import('@agoric/base-zone').Zone} zone
+ * @param {import('ava').ExecutionContext<unknown>} t
+ */
+const prepareAckWatcher = (zone, t) => {
+  return zone.exoClass('AckWatcher', undefined, packet => ({ packet }), {
+    onFulfilled(ack) {
+      t.is(ack, 'ack');
+      return 'fulfilled';
+    },
+    onRejected(reason) {
+      t.truthy(reason instanceof Error);
```

Comment:
`instanceof` is syntax that reliably returns a boolean, even of the right operand has a @@hasInstance method that returns a non-boolean. Therefore, if you wish, you could just say

```suggestion
      t.true(reason instanceof Error);
```

Up to you. No change requested.